### PR TITLE
[WIP] chore(user-data): update k8s scheduler to v1.0.7

### PR DESCRIPTION
**Pull request by mboersma**
Updates the k8s preview scheduler to what is considered the "known good version," 1.0.7, and turns k8s on for testing by default.

This is a work in progress, intended to ferret out what issues might exist that prevent k8s from being the preferred scheduler for Deis v1.x.

### CODE REVIEW

File: docs/customizing_deis/choosing-a-scheduler.rst [hunk 1]
```diff
@@ -11,9 +11,9 @@ The :ref:`scheduler` creates, starts, stops, and destroys each :ref:`container`
 of your app. For example, a command such as ``deis scale web=3`` tells the
 scheduler to run three containers from the Docker image for your app.
 
-Deis defaults to using the :ref:`fleet_scheduler`. Tech previews of schedulers based on
-:ref:`Kubernetes <k8s_scheduler>`, :ref:`Mesos with Marathon <mesos_scheduler>`, and
-:ref:`Swarm <swarm_scheduler>` are available for testing.
+Deis defaults to using the :ref:`fleet_scheduler`, but the :ref:`Kubernetes <k8s_scheduler>`
```
Comment (bacongobbler) [inline docs/customizing_deis/choosing-a-scheduler.rst]: "the Kubernetes" should be changed to "the Kubernetes scheduler" so it rolls off the tongue better :)

Comment (carmstrong) [inline docs/customizing_deis/choosing-a-scheduler.rst]: or "das kubernetes"

**Comment by helgi** [comment]
Should we bump this to 1.1 now that is out?

**Comment by krancour** [comment]
I am strongly in favor of moving in this direction... k8s as a default++.

**Pull request closed without merging**